builtin/method_io.py

```diff
@@ -232,6 +232,41 @@ def Call(self, rd):
         return value.Str(stdout_str)
 
 
+class CaptureOutputs(vm._Callable):
+    def __init__(self, mem, shell_ex):
+        # type: (state.Mem, vm._Executor) -> None
+        self.mem = mem
+        self.shell_ex = shell_ex
+
+    def Call(self, rd):
+        # type: (typed_args.Reader) -> value_t
+        unused = rd.PosValue()
+        cmd = rd.PosCommand()
+        # Allow us to not fail on error
+        fail = rd.NamedBool("fail", True)
```

Comment:
I think it might be OK to NEVER fail, because returning the status sort of implies that